bash_command_timer.sh

```diff
@@ -43,7 +43,9 @@ BCT_ENABLE=1
 #
 # If empty, disable colored output. Set it to empty if your terminal does not
 # support VT100 escape sequences.
-BCT_COLOR='34'
+#
+# The color of the output is set in the function BCTPreCommand
+# e.g. BCT_COLOR='34'
```

Comment:
Thanks for the PR! I like the idea a lot 👍 

I think it'd be nice if both colors could be customized here in the `SETTINGS` section instead of embedded below in the code - e.g. we could define `BCT_SUCCESS_COLOR` and `BCT_ERROR_COLOR` here, and use them below in the code.